doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -395,6 +396,10 @@ Cellular samples (renamed from nRF9160 samples)
     * The sample to remove redundant shadow updates for nRF Cloud.
     * Build instructions, board files, and DTC overlay file so that Wi-Fi scanning works for the nRF9161 DK and the nRF9160 DK.
 
+  * Fixed:
+
+    * Legitimate server side errors are not counted now as a reason to disconnect from and reconnect to the cloud.
```

Comment:
Can you elaborate on this? Maybe I missed it, but I didn't see any logic changes regarding disconnect due to server side errors. But if there is a server-side error ongoing, to me it seems valid to disconnect and try again later -- unless you are talking specifically about non-fatal server-side errors that don't require a reconnect for the link with nRFCloud to continue functioning?

Reply:
I'm talking about errors like getting a 4.04 error on a CoAP transfer indicating the resource requested could not be found.  There is no valid reason to disconnect from the cloud and reconnect, which wastes power and data.

The change is in message_queue.c line 134.

Currently MQTT transfers always return negative error codes or zero, so this changelog entry does not apply to that.

Reply:
I'll improve the comment.

Reply:
Ooh, gotcha. That delta totally flew under my radar. Thanks for the clarification